README.md

```diff
@@ -1,59 +1,42 @@
-# AI Legion: an LLM-powered autonomous agent platform
-
-A framework for autonomous agents who can work together to accomplish tasks.
-
-[![Discord](https://img.shields.io/discord/1095770840173383802?label=discord)](https://discord.gg/X9MkUEsEUC)
-
-## Setup
-
-You will need at least Node 10.
+### Install
 
 ```
 npm install
 ```
 
-Rename the `.env.template` file at the root of the project to `.env` and add your secrets to it:
+### Configure
+
+Create a `.env` file at the root of the project and add your secrets to it:
 
 ```
 OPENAI_API_KEY=... # obtain from https://platform.openai.com/account/api-keys
 # the following are needed for the agent to be able to search the web:
 GOOGLE_SEARCH_ENGINE_ID=... # create a custom search engine at https://cse.google.com/cse/all
 GOOGLE_API_KEY=... # obtain from https://console.cloud.google.com/apis/credentials
-AGENT_DELAY=... # optionally, a delay in milliseconds following every agent action
 ```
 
 You'll also need to enable the Google Custom Search API for your Google Cloud account, e.g. <https://console.cloud.google.com/apis/library/customsearch.googleapis.com>
 
-## Running
+## Run
 
 Start the program:
 
 ```
-npm run start [# of agents] [gpt-3.5-turbo|gpt-4]
+npm run start agents=[# of agents] model=[gpt-3.5-turbo|gpt-4] test=[true|false]
```

Comment:
This seems like the only change in this file that's relevant to this PR, can we revert the rest? In particular I don't understand why you removed the part at the top with the discord link or the action errors section.

Reply:
Ahh shite - that was unintentional - I had split our the readme to the parent readme for the rpo - which contained all the info like the disord linked and it had two links to the backend readme and the frontend readme.  This was just the backend readme.  It needs to be reintegrated with the monorepo readme